data/human/human missions.txt

```diff
@@ -7455,3 +7455,1836 @@ mission "Adelita 5"
+					goto surprise
+				`	(Get closer.)`
+				`	(Hail the pirates and use diplomacy.)`
+					goto diplomacy
+				`	(This is too dangerous. Leave.)`
+					flee
+
+			`	Getting as far as you can toward the base without compromising your position, you notice the base is surrounded by an array of laser turrets and ground-to-air missile systems, which makes further approach highly risky. Furthermore, the ships outside the base appear to be highly modified. You don't see any other entrance to the base, however; it looks like you'll have to force the pirates out, one way or another.`
+			`	It still seems as if the pirates don't see you. With a little movement, you could strafe the encampment and take out a fair few of the parked ships if you can avoid the turrets.`
+			choice
+				`	(Strafe the parked pirate vessels.)`
+					to display
+						"armament deterrence" > 0
+				`	(Hail the pirates and use diplomacy.)`
+					goto diplomacy
+				`	(This is too dangerous. Leave.)`
+					flee
+
+			action
+				set "diminished greenrock pirate fleet"
+			`	Carefully maneuvering into position, your ship flies out of the hills that hid it and lets loose a withering barrage over the parked pirate vessels. You manage to destroy a number of the ships while their shields are down, before the defense turrets force you to fly back up into space as the surviving ships sortie after you.`
+				launch
+
+			label diplomacy
+			action
+				set "big greenrock pirate fleet"
+				set "Timothy Radrickson 3b: diplomacy"
+			`	You broadcast a message to the ships parked in the encampment, declaring that you wish them no harm and inviting them to parley over the Rand convoy.`
+			`	Their only response is to launch their entire fleet after you. You're forced back into space to fight them.`
+				launch
+
+			label surprise
+			action
+				set "big greenrock pirate fleet"
+			`	You shunt your thrusters to full speed and fly into the valley, straight for the largest cluster of parked ships outside the blast doors.`
+			`	That is, until your sensors run hot with a multitude of approaching enemy warnings and missile locks. Without the protection of the hills that were hiding your ship, it looks like the pirates' early warning system detected you before you could get into position to attack. You're forced to turn back as an entire fleet of marauders fire up their engines and chase you into space.`
+				launch
+
+	on visit
+		dialog `There are still marauder ships flying overhead. Take off and defeat them before finding the fleet.`
+
+	npc kill
+		to spawn
+			has "big greenrock pirate fleet"
+		government "Pirate"
+		personality launching nemesis target plunders
+		fleet
+			names "pirate"
+			variant
+				"Marauder Falcon (Engines)"
+				"Falcon (Javelin)"
+				"Marauder Firebird (Weapons)"
+				"Firebird"
+
+				"Marauder Raven"
+				"Modified Argosy (Gatling)" 2
+				"Osprey (Missile)"
+
+				"Hawk (Plasma)"
+				"Quicksilver" 2
+				"Headhunter"
+				"Fury (Gatling)" 3
```

Comment:
Except for marauder should be capitalized?

Reply:
Elsewhere they use marauder with lower case elsewhere and the pirate ships aren't all Marauder variants, so I _think_ in this case they mean marauder in the general sense of being a synonym for raider/pirate rather than specifically Marauder ships in the Proper Noun sense.